core/core-load-paths.el

```diff
@@ -94,8 +94,8 @@
 ;;
 ;; This is based on `user-emacs-directory', not `spacemacs-start-directory',
 ;; because Spacemacs may be installed to a shared location and this directory
-;; and its children should be per-user.
-(defconst spacemacs-cache-directory
+;; and its children should be per-user. Allow customer define their own value.
```

Comment:
I'm a little confused by the word "customer" here, "user" here would be more idiomatic and consistent with other docs.  (FWIW I don't think this comment adds any value if there's a defcustom immediately below)